performance-test/conf/base/parameters.yml

```diff
@@ -0,0 +1,3 @@
+hook_delay: 0
+dataset_load_delay: 0
+file_save_delay: 0
```

Comment:
can we choose one name? either data_save_delay or file_load_delay.

Reply:
We could just call them "save_delay" and "load_delay" maybe?

Reply:
sounds good